[2023-04-26T15:17:39.121+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag_api.create_table_postgres manual__2023-04-26T15:17:30.223973+00:00 [queued]>
[2023-04-26T15:17:39.209+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag_api.create_table_postgres manual__2023-04-26T15:17:30.223973+00:00 [queued]>
[2023-04-26T15:17:39.214+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T15:17:39.220+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-26T15:17:39.232+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T15:17:39.383+0000] {taskinstance.py:1309} INFO - Executing <Task(PostgresOperator): create_table_postgres> on 2023-04-26 15:17:30.223973+00:00
[2023-04-26T15:17:39.393+0000] {standard_task_runner.py:55} INFO - Started process 2557 to run task
[2023-04-26T15:17:39.464+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weather_dag_api', 'create_table_postgres', 'manual__2023-04-26T15:17:30.223973+00:00', '--job-id', '146', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag_api.py', '--cfg-path', '/tmp/tmp3eoxmajl']
[2023-04-26T15:17:39.470+0000] {standard_task_runner.py:83} INFO - Job 146: Subtask create_table_postgres
[2023-04-26T15:17:40.228+0000] {task_command.py:389} INFO - Running <TaskInstance: weather_dag_api.create_table_postgres manual__2023-04-26T15:17:30.223973+00:00 [running]> on host d48b88797b3b
[2023-04-26T15:17:42.201+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_dag_api
AIRFLOW_CTX_TASK_ID=create_table_postgres
AIRFLOW_CTX_EXECUTION_DATE=2023-04-26T15:17:30.223973+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-26T15:17:30.223973+00:00
[2023-04-26T15:17:42.204+0000] {sql.py:254} INFO - Executing:  CREATE TABLE IF NOT EXISTS measurements (
			timestamp TIMESTAMP,
			temp FLOAT,
			precipitation FLOAT,
			wind_speed FLOAT,
			location TEXT);
[2023-04-26T15:17:42.303+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-04-26T15:17:43.666+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-04-26T15:17:43.680+0000] {sql.py:375} INFO - Running statement:  CREATE TABLE IF NOT EXISTS measurements (
			timestamp TIMESTAMP,
			temp FLOAT,
			precipitation FLOAT,
			wind_speed FLOAT,
			location TEXT);, parameters: None
[2023-04-26T15:17:43.721+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=weather_dag_api, task_id=create_table_postgres, execution_date=20230426T151730, start_date=20230426T151739, end_date=20230426T151743
[2023-04-26T15:17:43.811+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-26T15:17:43.948+0000] {taskinstance.py:2596} INFO - 5 downstream tasks scheduled from follow-on schedule check
